src/components/AppLayout/Header/index.tsx

```diff
@@ -59,7 +54,7 @@ const HeaderComponent = (): React.ReactElement => {
     return (
       <UserDetails
         connected={available}
-        onDisconnect={onDisconnect}
+        onDisconnect={onboard().walletReset}
```

Comment:
With this syntax it might or might not work depending on whether this method uses `this`.

`object.methodCall()` isn't the same as `const callback = object.methodCall; callback()`.

Reply:
I've checked the onboard source code, it's not an actual instance method, just some exported function. So it should be fine.